setup.py

```diff
@@ -40,11 +34,5 @@ def get_long_description():
         'Programming Language :: Python :: 3',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    install_requires=[
-        'requests',
-        'more-executors',
-        'six',
-        'monotonic',
-        'edgegrid-python',
-    ],
+    install_requires=open('requirements.txt').readlines(),
```

Comment:
Non blocking comment, I would go with open().read().splitlines() to avoid the new line characters.

Reply:
Fixed.